src/config/templates.toml

```diff
@@ -26,13 +26,27 @@ label(if(current_working_copy, "working_copy"),
 )
 '''
 
+op-log = '''
+label(if(current_head, "head"),
+  separate(" ",
+    id.short(),
+    user,
+    format_operation_duration(time),
+  )
+  "\n"
+  description.first_line() "\n"
+  tags
+)
+'''
+
 # Defined as alias to allow 'log -T show'
 show = 'show'
 
 [template-aliases]
 'description_placeholder' = 'label("description", "(no description set)")'
 
 # Hook points for users to customize the default templates:
+'format_operation_duration(time_range)' = 'time_range.relative()'
```

Comment:
I feel like we should change the default template to be something like this:
```
'format_time_range(time_range)' = 'format_timestamp(time_range.start()) " - " format_timestamp(time_range.end()) '
```

We should also let the user get the duration form by something like this:
```
'format_time_range(time_range)' = 'format_timestamp(time_range.start()) ", lasted " time_range.duration()'
```

What do you think? Maybe this is already in your plans?

Reply:
Not considered that, but sounds good. I'll add TODO comment.

Reply:
This was done in this PR. The default template ended up with
```
'format_time_range(time_range)' = '''
  time_range.start().ago() label("time", ", lasted ") time_range.duration()'''
```

It's weird that ", lasted " is labeled as time, but otherwise the message doesn't look like a sentence.